helpers/btfinfo.go

```diff
@@ -0,0 +1,118 @@
+package helpers
+
+import (
+	"bufio"
+	"errors"
+	"fmt"
+	"golang.org/x/sys/unix"
+	"os"
+	"strings"
+)
+
+func fileExists(path string) bool {
+	if _, err := os.Stat(path); err == nil {
+		return true
+	}
+	return false
+}
+
+// fileGrepUnique: returns first 'needle' string found in a file 'path'.
+func fileGrepUnique(path string, needle string) string {
+	file, _ := os.Open(path)
+	defer func(file *os.File) {
+		_ = file.Close()
+	}(file)
```

Comment:
Why not simply deferring file.Close()?

Reply:
this was added automatically by goland ... argh